docs/geps/gep-03.rst

```diff
@@ -93,7 +93,36 @@ entry is added.
                 Shares of income which do not lead to tapering of Arbeitslosengeld II
                 benefits.
 
-3. The YYYY-MM-DD key(s)
+3. The ``unit`` key informs on the unit of the values (Euro or DM if monetary, or some share).
```

Comment:
I think Eric is right and we should add this information. I would use a different key though. I suggest we keep all that under the type key, which is, by the way, missing here so far. I can add it in the coming days. Possible values for type would then be: Euro, DM, share, percentage, piecewise...

Reply:
I am afraid I do not follow your conversation. I do think we should add units, but I do not understand what the point of contention is / what the options are. Sorry...

Reply:
Sorry, this is related to a private discussion on zulip. Thought this would wrap it up. So we have the `type` key. So far we only used it for piecewise functions. I suggest that we just keep this key and add there our information on units. But actually not a strong opinion. If you think we should have type, unit and time, I will adjust!

Reply:
Thanks! 

I do have a strong opinion here: No key overloading, please! 

I.e., there is a lot of value in knowing exactly what is in a key without recurring to external information. If we were using the type key, we would need to know what kind of parameter we are talking about, which defies most of its purpose. 

Actually, we should make the unit an ENUM-style key (or categorical, in Pandas-speak), i.e., it should only be able to take on a number of specified values. E.g., DM, EUR, share, percent, ..., you name it.